app/components/vault/VaultMasterAccounts.js

```diff
@@ -0,0 +1,136 @@
+// @flow
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { smColors } from '/vars';
+import { DropDown, Tooltip } from '/basicComponents';
+
+const isDarkModeOn = localStorage.getItem('dmMode') === 'true';
+
+const DetailsRow = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-bottom: 20px;
+`;
+
+const DetailsText = styled.div`
+  font-size: 16px;
+  line-height: 20px;
+  color: ${isDarkModeOn ? smColors.white : smColors.realBlack};
+`;
+
+const Dots = styled.div`
```

Comment:
make this basicComponent as it repeats itself multiple times

Reply:
done